### ISSUE TITLE
[node-gyp] Rebuild error back(?) on multiple macs. 

### ISSUE THREAD

**Issue by BorisKourt** [sandeepmistry/noble#209]
Hello,
I have seen a few issues and commits related to this recently but can only find  #206  active now, and am not sure if it is related. 

I remember having this issue a few months ago, and that it went away for a long while.

Tested it on a bunch of 10.10.3 Macs. Here is the npm error message:

```sh
boriskourt@Boriss-MacBook-Air ~/D/s/xsfrj> npm install noble --save
/
> xpc-connection@0.1.3 install /Users/boriskourt/node_modules/noble/node_modules/xpc-connection
> node-gyp rebuild

  CXX(target) Release/obj.target/binding/src/XpcConnection.o
  SOLINK_MODULE(target) Release/binding.node

> noble@1.0.1 install /Users/boriskourt/node_modules/noble
> node-gyp rebuild

  LINK(target) Release/hci-ble
Undefined symbols for architecture x86_64:
  "_main", referenced from:
      start in crt1.10.5.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Release/hci-ble] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Darwin 14.3.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/boriskourt/node_modules/noble
gyp ERR! node -v v0.12.7
gyp ERR! node-gyp -v v2.0.1
gyp ERR! not ok
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "noble" "--save"
npm ERR! node v0.12.7
npm ERR! npm  v2.12.1
npm ERR! code ELIFECYCLE

npm ERR! noble@1.0.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the noble@1.0.1 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the noble package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls noble
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/boriskourt/Documents/sandbox/xsfrj/npm-debug.log
```
I get a similar one on a fresh setup as well.

Do you want the `npm-debug.log`?

Thanks

**Comment by sandeepmistry** [comment]
@BorisKourt the fix (#205) was not published to npm.

It's included in the v1.0.2 release that I've published now.

**Comment by BorisKourt** [comment]
Thank you!
